icecream/icecream.py

```diff
@@ -173,6 +197,8 @@ def __init__(self, prefix=DEFAULT_PREFIX,
         self.includeContext = includeContext
         self.outputFunction = outputFunction
         self.argToStringFunction = argToStringFunction
+        self.passWidthParam = supports_param(self.argToStringFunction)
+        self.lineWrapWidth = detect_terminal_width(self.prefix, DEFAULT_LINE_WRAP_WIDTH)
```

Comment:
This still doesn't account for the width of `argPrefix`, or changing the prefix with `configureOutput`.

Reply:
Good point, will fix.